mina-payout-reports/templates/deployment-api.yaml

```diff
@@ -0,0 +1,143 @@
+            - name: POSTGRES_PAYOUT_PASSWORD
+              value: {{ .payoutsDB.password | quote }}
+            - name: POSTGRES_ARCHIVE_HOST
+              value: {{ .archiveDB.host | quote }}
+            - name: POSTGRES_ARCHIVE_PORT
+              value: {{ .archiveDB.port | quote }}
+            - name: POSTGRES_ARCHIVE_USER
+              value: {{ .archiveDB.user | quote }}
+            - name: POSTGRES_ARCHIVE_PASSWORD
+              value: {{ .archiveDB.password | quote }}
+            - name: POSTGRES_ARCHIVE_DB
+              value: {{ .archiveDB.name | quote }}
+            - name: SENDGRID_API_KEY
+              value: {{ .sendgridToken | quote }}
+            - name: SPREADSHEET_CREDENTIALS_JSON
+              value: /app/mina_payout/api/gcloud/gcp-sa.json
+            - name: CONTACT_DETAILS_SPREADSHEET_NAME
+              value: {{ .contactDetailsSpreadsheetName | quote }}
+            - name: GCS_BUCKET_NAME
+              value: {{ .stakingLedgersBucket | quote }}
+            - name: BP_WALLET_MAPPING_URL
+              value: {{ .walletMappingSpreadsheetUrl | quote }}
+            - name: BP_WALLET_MAPPING_TAB
+              value: {{ .walletMappingSpreadsheetTab | quote }}
+            - name: AWS_BUCKET_NAME
+              value: {{ .s3bucket | quote }}
+            - name: AWS_REGION
+              value: {{ .aws.region | quote }}
+            - name: AWS_ACCESS_KEY_ID
+              value: {{ .aws.accessKeyID }}
+            - name: AWS_SECRET_ACCESS_KEY
+              value: {{ .aws.secretAccessKey }}
+            - name: SERVER_ALLOWED_ORIGINS
+              value: "*"
+            {{- if .extraEnvVars }}
+            {{- toYaml .extraEnvVars | nindent 12 }}
+            {{- end }}
+          volumeMounts:
+            - name:  gcp-sa
+              mountPath: /app/mina_payout/api/gcloud
+          ports:
+            - name: http
+              containerPort: {{ .service.port }}
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /health
+              port: http
+          readinessProbe:
+            httpGet:
+              path: /health
+              port: http
+          resources:
+            {{- toYaml .resources | nindent 12 }}
+      {{- with .nodeSelector }}
+      nodeSelector:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .affinity }}
+      affinity:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .tolerations }}
+      tolerations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      volumes:
+        - name: entrypoint-initdb
+          configMap:
+            name: {{ include "mina-payout-reports.fullname" . }}-initdb
+            defaultMode: 0777
+            items:
+              - key: "entrypoint-initdb.sh"
+                path: "entrypoint-initdb.sh"
+        - name: gcp-sa
+          secret:
+            secretName: {{ include "mina-payout-reports.fullname" . }}
```

Comment:
**suggestion:** I was unable to render the chart until applying this change:

```suggestion
            name: {{ include "mina-payout-reports.fullname" $ }}-initdb
            defaultMode: 0777
            items:
              - key: "entrypoint-initdb.sh"
                path: "entrypoint-initdb.sh"
        - name: gcp-sa
          secret:
            secretName: {{ include "mina-payout-reports.fullname" $ }}
```

Reply:
yes, that's because `with` changed the scope so `.` in that scope does not work and $ is needed.